backend/app/views/spree/admin/tax_rates/index.html.erb

```diff
@@ -55,7 +83,7 @@
         <td><%=tax_rate.included_in_price? ? t('spree.say_yes') : t('spree.say_no') %></td>
         <td><%=tax_rate.show_rate_in_label? ? t('spree.say_yes') : t('spree.say_no') %></td>
         <td><%=tax_rate.expires_at.to_date.to_s(:short_date) if tax_rate.expires_at %></td>
-        <td><%=tax_rate.calculator.to_s %></td>
+        <td><%=tax_rate.calculator && tax_rate.calculator.class.model_name.human %></td>
```

Comment:
What do you think about having it in a helper?

Reply:
I prefer it that way. This is established Rails.

What do you hope to gain from a helper?

Reply:
I like to avoid logic in the template. It helps prevent duplication and makes templates more readable. But that's non-blocking. As you said, it's established Rails, although I disagree with it :slightly_smiling_face:

Reply:
I see. I would not call this logic in the template. It's a translation with a safe guard (that might be unnecessary, but the test failed without).

Also Solidus being an extendable framework I would even argue against helpers at all. Presenters and templates are way easier to overwrite and extend. 

But that's another discussion. Thanks for the review.